game-app/game-core/src/main/java/org/triplea/sound/SoundOptions.java

```diff
@@ -46,7 +44,8 @@ private static void showSoundOptions(final JComponent parent) {
             JOptionPane.DEFAULT_OPTION,
             null,
             new Object[] {ok, selectAll, selectNone, cancel});
-    pane.createDialog(parent, "Sound Options").setVisible(true);
+    // use default frame to avoid button action dependency to itself
+    pane.createDialog(null, "Sound Options").setVisible(true);
```

Comment:
```suggestion
    // use default frame to avoid button action dependency to itself
    JFrame defaultFrame = null;
    pane.createDialog(defaultFrame, "Sound Options").setVisible(true);
```